framework/lib/components/createable-select-dropdown/types.ts

```diff
@@ -1,11 +1,16 @@
 import { Variant } from 'chakra-react-select/dist/types/types'
+import { ComponentProps } from 'react'
+import { CreatableSelectComponent } from 'chakra-react-select'
 import { Option } from '../select'
 
-export interface CreationOption extends Option {
+export interface CreationOption <T extends string = string> extends Option<T> {
   isCreation: boolean
 }
 
-export interface CreatableSelectDropdownProps<T extends string = string> {
+export type CreationOptionValue = 'add_field'
```

Comment:
you can use this type on the value constant in the creatable-select-dropdown component.